### PR TITLE
BZ-1641584: Fixing incorrect values for certfile and cafile

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 3.11 only
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1641584
Preview Link: https://docs.openshift.com/container-platform/3.11/install_config/certificate_customization.html#ansible-configuring-custom-certificates
QE ack required: @wangke19

### CODE REVIEW

**Review by skrthomas — APPROVED** [review]
/lgtm

File: install_config/certificate_customization.adoc [hunk 1]
```diff
@@ -61,7 +61,7 @@ openshift_hosted_router_certificate={"certfile": "/path/on/host/to/app-crt-file"
 Example parameters for a master API certificate:
 ----
 openshift_master_overwrite_named_certificates=true
-openshift_master_named_certificates=[{"names": ["master.148.251.233.173.nip.io"], "certfile": "/home/cloud-user/master-bundle.cert.pem", "keyfile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem", "cafile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem"}]
+openshift_master_named_certificates=[{"names": ["master.148.251.233.173.nip.io"], "certfile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem", "keyfile": "/home/cloud-user/master-bundle.cert.pem", "cafile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem"}]
```
Comment (wangke19) [inline install_config/certificate_customization.adoc:64]: @sagidlow the value `/home/cloud-user/master-bundle.cert.pem` is for parameter `cafile `,  `/home/cloud-user/master.148.251.233.173.nip.io.key.pem` is for `keyfile`, certfile and keyfile is a pair.

File: install_config/certificate_customization.adoc [hunk 2]
```diff
@@ -61,7 +61,7 @@ openshift_hosted_router_certificate={"certfile": "/path/on/host/to/app-crt-file"
 Example parameters for a master API certificate:
 ----
 openshift_master_overwrite_named_certificates=true
-openshift_master_named_certificates=[{"names": ["master.148.251.233.173.nip.io"], "certfile": "/home/cloud-user/master-bundle.cert.pem", "keyfile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem", "cafile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem"}]
+openshift_master_named_certificates=[{"names": ["master.148.251.233.173.nip.io"], "certfile": "/home/cloud-user/master-bundle.cert.pem", "keyfile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem", "cafile": "/home/cloud-user/master-bundle.cert.pem"}]
```
Comment (wangke19) [inline install_config/certificate_customization.adoc:64]: Change "certfile": "/home/cloud-user/master-bundle.cert.pem" to "/home/cloud-user/master.148.251.233.173.nip.io.key.pem".

**Pull request merged**